app/graphql/types/pageable.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module Types
+  module Pageable
```

Comment:
Is there any benefit to making this extend ActiveSupport::Concern? (if that would work)

Reply:
I couldn't think of one but if you can?

Reply:
I guess it's the same as asking "why ever use ActiveSupport::Concern in a rails app at all?"

The reason the question crossed my mind is because we've been using it for included modules in other places.  We don't have to, but I just thought I'd put it out there.  Up to you.

Reply:
Thinking more about it, using `ActiveSupport::Concern` is a little DRYer, right?